lnrpc/lightning.proto

```diff
@@ -1091,6 +1091,18 @@ message LightningAddress {
     string host = 2;
 }
 
+enum CoinSelectionStrategy {
+    // Use the coin selection strategy defined in the global configuration
+    // (lnd.conf).
+    STRATEGY_USE_GLOBAL_CONFIG = 0;
```

Comment:
should we call it it 'default'? im open to being wrong here if other reviewers disagree 👍

Reply:
nvm - I agree with Oli that being more explicit here is good 👍